backend-app/utils/authorization/roles/role.ts

```diff
@@ -2,7 +2,7 @@ import AppError from '@utils/app_error';
 import roleModel from '@models/user/role_model';
 import Actions from '@constants/actions';
 
-interface RoleData {
+export interface RoleData {
```

Comment:
Can you tell me why you exported this?
because if it is needed, we can move it to interfaces instead

Reply:
I have exported this for creating dtos but this doesnt have the name prop so i have to create a new interface then i forgot to remove the export keyword. i will remove this in the next commit.

Reply:
oh, nice 👌gj